test/jdk/javax/swing/text/BoxView/bug6494356.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2011, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/* @test
+ * @bug 6494356
+ * @key headful
+ * @summary Test that BoxView.layout() is not called with negative arguments
+ * @run main bug6494356 bug6494356.html
```

Comment:
Can the test be converted to non-applet based?

Reply:
It 's not applet-based. The HTML file is an argument to the test, it is the file that's loaded into `JEditorPane`.

Reply:
Oh, my bad, didn't notice it.

Reply:
Yeah, it's a bit confusing…